doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+
++--------------+---------------------------------------------------------------+
+| Plutus contract testing model                                                |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+| Dependencies | plutus-ledger, plutus-ledger-api, plutus-ledger-constraints,  |
+|              | plutus-contract                                               |
++--------------+---------------------------------------------------------------+
+| State        | Maintained and worked on. Developed by QuviQ.                 | 
++--------------+---------------------------------------------------------------+
+
+11. Plutus use case examples
+------------------------------------------
+
+`Plutus use case examples <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-use-cases>`_ 
+contains hand-written examples for the use cases we currently have. The primary 
+examples are: 
+
+   * Auction, 
+   * Crowdfunding, 
+   * Game, 
+   * GameStateMachine, and 
+   * Vesting. 
+
+For each plutus application use case, we provide test scenarios (or test cases) 
+with and without the :hsmod:`Plutus.Contract.Test.ContractModel`.
+
++--------------+---------------------------------------------------------------+
+| Plutus use case examples                                                     |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-use-cases``                              |
++--------------+---------------------------------------------------------------+
+| Dependencies | plutus-ledger, plutus-ledger-api, plutus-ledger-constraints,  |
+|              | plutus-contract                                               |
++--------------+---------------------------------------------------------------+
+| State        | Maintained. The examples are used as test cases.              |
+|              | While they work in the plutus-contract emulator, they         |
+|              | are not guaranteed to work on the actual Cardano network.     | 
++--------------+---------------------------------------------------------------+
+
+12. PAB (Plutus application backend)
+------------------------------------------
+
+`PAB <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-pab>`_ 
+is a web server library for managing the state of Plutus contract instances. 
+The PAB executes the off-chain component of Plutus applications. It manages 
+application requests to the wallet backend, the Cardano node and the chain-index. 
+PAB stores the application state and offers an HTTP REST API for managing application 
+instances. 
+
+PAB wraps the contracts built with plutus-contract. It is the central point of 
+contact for everything, integrating all the Cardano components. 
```

Comment:
It does integrate a lot but not "all", e.g. not Marconi
Could say "integrating many Cardano components"